build.sc

```diff
@@ -540,6 +540,7 @@ object dev extends MillModule{
     } else {
       os.perms.set(outputPath, "rwxrwxrwx")
     }
+    T.ctx().log.info(s"Launcher script was written to $outputPath")
```

Comment:
We shouldn't need to log this path here; anyone who wants it can already use `mill show dev.launcher` to see the full path to the script, or the value returned by any other task they wish to run

Reply:
Ok, makes sense